components/clinical/hospital-stay-summary/src/organisms/admission-summary.tsx

```diff
@@ -0,0 +1,66 @@
+import { HTMLAttributes, FC } from 'react'
+import styled from '@emotion/styled'
+
+import { Encounter, EncounterStatusCode, Maybe } from '@ltht-react/types'
+import { PeriodSummary } from '@ltht-react/type-summary'
+
+import { BANNER_COLOURS } from '@ltht-react/styles'
+import ServiceProvider from '../atoms/hospital-stay-service-provider'
+import Description from '../atoms/hospital-stay-description'
+import Redacted from '../molecules/hospital-stay-redacted'
+import AdmissionLengthOfStay from '../atoms/admission-length-of-stay'
+
+const StyledSummary = styled.div<IStyledSummaryProps>`
+  display: flex;
+  justify-content: center;
+  ${({ status }) =>
+    status === EncounterStatusCode.InProgress &&
```

Comment:
I think this could lead to printing out `false` in the generated css when the status is not `InProgress`? I.e.

```css
display:flex;
justify-content: center;
false
```

Reply:
change done